tests/test_client.py

```diff
@@ -263,6 +264,36 @@ def test_with_invalid_request():
         Client(should_validate_auth=False).submit(request)
 
 
+@responses.activate
+def test_request_has_user_agent_headers():
+    collection = Collection('foobar')
+    request = Request(
+        collection=collection,
+    )
+    responses.add(
+        responses.GET,
```

Comment:
You should probably add a case for POST requests as well.

Reply:
Yeah, I realized that after sending out PR ... if it's easy for you, could you point me to a minimal test case using POST?

Reply:
Alright, it's actually quite easy. I just added test case for POST and commit the change.